elliottlib/errata_async.py

```diff
@@ -0,0 +1,116 @@
+import asyncio
+import base64
+import logging
+from typing import Dict, List, Set, Union
+from urllib.parse import quote, urlparse
+
+import aiohttp
+import gssapi
+from kobo.rpmlib import parse_nvr
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AsyncErrataAPI:
+    def __init__(self, url: str):
+        self._errata_url = urlparse(url).geturl()
```

Comment:
Any reason constans.py is not consumed here?

Reply:
A client instantiating AsyncErrataAPI usually get Errata URL from erratatool.yml in ocp-build-data. I don't want to couple things unrelated to API calls here.